client/i18n/README.md

```diff
@@ -0,0 +1,117 @@
+This project uses `i18next` with `react-i18next` to provide internationalization of content. Human readable content should not be used directly in React components. Instead, they should be looked up using `react-i18next` - which will ensure the content is localized. The library also supports interpolation of dynamic values and formatting of the output. `eslint-plugin-i18next` is used to ensure that JSX cannot be written that contains text without a translation.
```

Comment:
Uber picky - worth adding a title for nicer formatting (`# internationalisation (i18n)`)?